registrations/tasks.py

```diff
@@ -1543,3 +1543,4 @@ def request_to_jembi_api(url, json_doc):
         verify=False,
     )
     r.raise_for_status()
```

Comment:
You can probably remove this, we don't want to `raise_for_status` twice.

Reply:
okay thanks